src/compiler/commandLineParser.ts

```diff
@@ -996,9 +996,9 @@ namespace ts {
         return { options, errors };
     }
 
-    export function convertTypingOptionsFromJson(jsonOptions: any, basePath: string, configFileName?: string): { options: TypingOptions, errors: Diagnostic[] } {
+    export function convertTypeAcquisitionFromJson(jsonOptions: any, basePath: string, configFileName?: string): { options: TypeAcquisition, errors: Diagnostic[] } {

```

Comment:
question to @mhegazy - do we consider this a breaking change since this API was public in 2.0. this is also related to name of section in config file